examples/ddp2pipe/ddp2pipe.py

```diff
@@ -0,0 +1,299 @@
+import argparse
```

Comment:
This example is great! Could you add a comment in the preamble explaining the flow?